packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx

```diff
@@ -66,6 +66,10 @@ export const AddressInput = ({ value, name, placeholder, onChange, disabled }: C
     onChange(ensAddress);
   }, [ensAddress, onChange, debouncedValue]);
 
+  useEffect(() => {
+    setEnteredEnsName(undefined);
```

Comment:
since you added this `useEffect`, `handleChange` function can be deleted and use `onChange={onChange}` for `InputBase` props

Reply:
oh yes!! great catch https://github.com/scaffold-eth/scaffold-eth-2/pull/926/commits/a090b57b94d17ea431794005cebbc4d802522944